client/src/themes/theme.js

```diff
@@ -6,9 +6,20 @@ export const theme = createMuiTheme({
     fontSize: 12,
     h1: {
       // could customize the h1 variant as well
-    }
+      fontSize: "24px",
+      fontWeight: "bold",
+      textAlign: "right",
+      color: "black",
+    },
+    subtitle1: {
+      fontWeight: "bold",
+    },
   },
   palette: {
-    primary: { main: "#DF1B1B" }
-  }
+    // primary: { main: "#DF1B1B" },
```

Comment:
we can remove all commented out code before merging, can you remove this in your next PR.

Reply:
Will do